learning_observer/learning_observer/auth/social_sso.py

```diff
@@ -210,6 +248,34 @@ async def _process_student_documents(student):
             await _process_student_documents(student)
     # TODO saved skipped doc ids somewhere?
 
+async def _canvas(request):
```

Comment:
This is a horrible function name. A function name should describe what this does. `_canvas` is a noun. Does this DOS canvas servers? Grab rosters? Return the IP of our canvas instance? No way to know.

Reply:
I've fixed the function name